.github/workflows/ci-maven.yaml

```diff
@@ -156,20 +156,24 @@ jobs:
       matrix:
         java: [ 11, 17 ]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: actions/setup-java@v3
         with:
           java-version: ${{ matrix.java }}
           distribution: temurin
-      - run: java -version
+      - run: mvn -version
       - name: Install maven modules
         run: |
           mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip
 
-      - name: Install system dependencies
+      - name: Showcase golden tests
```

Comment:
Would this need to be omitted from https://github.com/googleapis/gapic-generator-java/blob/main/.github/workflows/downstream-native-image.yaml as well?